endpoint_cache_preheat/models/endpoint_mixin.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2025 Camptocamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import logging
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class EndpointMixin(models.AbstractModel):
+
+    _inherit = "endpoint.mixin"
+
+    cache_preheat = fields.Boolean(
+        string="Automatic cache pre-heat",
+        help="If checked, cache will be pre-heated by a cron "
+        "according to the selected policy. "
+        "Cache generation will be done in a queue job.",
+    )
+    cache_preheat_ts = fields.Datetime(readonly=True)
```

Comment:
```suggestion
    cache_preheat_timestamp = fields.Datetime(readonly=True)
```

Reply:
I've improved the label

Reply:
Yeah, it's the field name itself that could be more readable :smile: